src/content/platform-overview/security/cluster-security/domain-allowlist/index.md

```diff
@@ -25,18 +25,26 @@ Below is a list of the external domains we require access to for our clusters to
 - amazonaws.com
     - domains:
         - `*.amazonaws.com`
+        - `ec2.eu-west-2.amazonaws.com`
+        - `sts.eu-central-1.amazonaws.com`
     - AWS services are used for a variety of tasks, such as etcd backup storage.
 - azurecr.io
     - domains:
         - `giantswarm.azurecr.io`
+        - `giantswarmpublic.azurecr.io`
     - Container images are hosted on Azure Container Registry.
+- auth0.com
+    - domains:
+        - `giantswarm.eu.auth0.com`
+    - Used to secure access to Grafana and Prometheus.
 - cloudfront.net
     - domains:
         - `*.cloudfront.net`
     - Operators may pull from sites behind Cloudfront.
 - docker.com
     - domains:
         - `*.docker.com`
+        - `production.cloudflare.docker.com`
```

Comment:
Let's add a rationale for this one, so that we have the chance to remove it once not needed anymore